### PR TITLE
Feat/cta preview card

### PR DESCRIPTION
**Pull request by nicmak**
## Related issues

https://github.com/telus/tds-community/issues/234

## Description

- Adding a CTA Text in PreviewCard

## Checklist before submitting pull request

- Commits follow our [Developer Guide](https://github.com/telus/tds-community/blob/chore/template-hints/.github/CONTRIBUTING.md)
- [x] New code is unit tested
- [x] make sure visual and accessibility tests pass
- [x] make sure code builds

### CODE REVIEW

File: packages/PreviewCard/PreviewCard.jsx [hunk 1]
```diff
@@ -31,6 +31,10 @@ const BoxContainer = styled.div`
   display: inline-block;
 `
 
+const HrefTextSmall = styled.span`
+  font-size: 14px;
```
Comment (sketchidea) [inline packages/PreviewCard/PreviewCard.jsx]: Can you use rems instead of pixels for sizes.

File: packages/PreviewCard/PreviewCard.jsx [hunk 2]
```diff
@@ -83,8 +87,16 @@ const PreviewCard = ({ header, image, body, footer, href, linkComponent, ...rest
...
           )}
-          <Box vertical={hasHeaderOrFooter ? 3 : undefined}>
+          <Box
+            vertical={hasHeaderOrFooter ? 3 : undefined}
+            between={hasHeaderOrFooter ? 3 : undefined}
+          >
             <P alt={body}>{newBody}</P>
+            {hrefText && (
+              <ChevronLink href={href}>
+                <HrefTextSmall>{hrefText}</HrefTextSmall>
```
Comment (sketchidea) [inline packages/PreviewCard/PreviewCard.jsx]: can we rename hrefText to linkText.

Also, can we not use TDS Paragraph for font sizing? You should be able to wrap ChevronLink in a Paragraph and then ditch HrefTextSmall.

```jsx
<Paragraph>
  <ChevronLink href={href}>{linkText}</ChevronLink>
</Paragraph>

File: packages/PreviewCard/PreviewCard.jsx (hunk 1 above)
Comment (nicmak) [reply packages/PreviewCard/PreviewCard.jsx]: ok

File: packages/PreviewCard/PreviewCard.jsx (hunk 2 above)
Comment (nicmak) [reply packages/PreviewCard/PreviewCard.jsx]: ok

**Comment by bnadeem-telus** [comment]
This has been tested and is looking good.

**Comment by sketchidea** [comment]
Closing, as per recommendations

**Pull request closed without merging**